samples/azure/shared-host/CloudServicesHost_7/Sender/Sender.csproj

```diff
@@ -93,7 +93,9 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="app.config" />
-    <None Include="packages.config" />
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
```

Comment:
no, but VS adds it randomly. it is a known bug. i have given up removing it manually